module_utils/network_lsr/argument_validator.py

```diff
@@ -457,6 +457,11 @@ def __init__(self, name, family=None, required=False):
         self.family = family
 
     def _validate_impl(self, value, name):
+        # Note that ArgValidatorIPAddr accepts both a string and
+        # a dictionary (with "address" and "prefix") as input.
+        #
+        # But the validation will always unfold this into a dictionary
+        # containing "address", "family", ad "prefix".
```

Comment:
Since it is a description of the class, it seems like it would fit better in a docstring of the class. What do you think?

Reply:
There could be a class docstring. This comment was not intended to be a documentation of the class. It's a note about some special behavior that happens right here, during `validate()`. I reader of the code might be interested, how `validate()` behaves, that's why the note is there. For that aspect, I wouldn't want to look at the class docstring.

Also the docstring of the class should describe that this is in general a validator for an IP address, and the forms in which it accepts the input and normalizes the output. I am not strongly against such a documentation (if somebody wants to add it), but I'd rather read the code (that is hopefully easy to read and has a note/comment about the pitfalls/specialties), than a prosaic description about it.

By that argument this entire commit shouldn't be there. I'll drop it.